packages/web/src/ui/umd.js

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
Ok so I looked closer into this : 
- very nice work doing a good acceptable solution mostly re-using existing stuff as much as possible
- for the api itself, I am not so sure to be honest, it does the job well, BUT
 it is both similar AND  really is very different from the thing planned for V2 https://github.com/jscad/OpenJSCAD.org/blob/f543b50678df4eec32b32a5ae421264099ad86f9/packages/web/src/index.js#L13
and barebones init example here : https://github.com/jscad/OpenJSCAD.org/blob/f543b50678df4eec32b32a5ae421264099ad86f9/packages/web/demo.js

 to be more clear you create & use it something like 
```javascript
const jscadInst1 =  makeJscad(domElement, options)
// jscadInst is now a callable, that you can use as a function (pseudocode as it is not finalized !)
jscadInst1({cmdType: 'loadDesign', uri: 'http:///blabla'})
jscadInst1({cmdType: 'abort'})
// some async things would likely we awaitable 
await jscadInst1({cmdType: 'loadDesign', uri: 'http:///blabla'})

```
I am honestly not 100% sure:  if V1 was not 'end of maintenance' (but still living), I would not hesitate to say ok for your api ? 
I am not stuck on this, but perhaps you have some ideas on how to reconcile this ? 
Or we just say 'ah whatever, good to have an umd build for V1 , we have warned that V2 will break a lot of APIs, sorry, but deal with it ?'

Reply:
I'm not sure what's planned, but the `makeJscad` function only uses two things off of options object (name & logging).  If you have ideas for the other option names, I'll be happy to make it reflect that.  Is there a file you'd like me to use as a template for the processor and viewer options?

I'm really keen on getting to the v2 API since we will be able to use regl instead of lightgl or threejs, but I assume most of the options will be similar.

I'm sure there will also be other breaking changes, so maybe just some caveats in the release notes should be ok?

Reply:
There will not be any fundamental changes, except array use instead of x.y.z / rgba, and generally finer grained control over camera etc , but it does not make much sense to put these into V1 since they would not do anything :) 

additional options (not complete, might still change etc, but close to final) 
```
viewer: {
  camera: {
   near: 1,
   far: 18000,
   up: [0, 0, 1],
   position: [450, 550, 700],
   target: [0, 0, 0],
   fov: Math.PI / 4,
   aspect: 1,
   projectionType: 'perspective'
},
 controls: {
  limits: {
    minDistance: 0.01,
    maxDistance: 10000
  },
  drag: 0.27, // Decrease the momentum by 1% each iteration
  EPS: 0.000001,
},
  rendering: {
        background: [1, 1, 1, 1],
        meshColor: [0, 0.6, 1, 1]
      },
      grid: {
        show: false,
        color: [1, 1, 1, 0.1],
        size: [200, 200],
      },
      axes: {
        show: true
      },  
     autorotate: {
       enabled: true,
       speed: 1.0
     }
  }
}
```

Reply:
and you are right, let's deal with the changes between V1 and V2 later

Reply:
@kaosat-dev maybe it’s time for a V2 branch.... which @johnwebbcole can start contributing enhancements towards.

😃